### DISCUSSION TITLE
My profile shows a 404 error to others but is accessible to me when logged in

### DISCUSSION THREAD

**Discussion by Muhammad-kaleem-dev** [community/community#146922] [Other Feature Feedback, Questions, & Ideas]
### profile 

Question

### Body

I am experiencing an issue with my GitHub account. While I can log in and access my account without any problems, my profile is not visible to others. When I or anyone else tries to visit my profile at [url](https://github.com/KaleemNeslit) they encounter a 404 error.
I have verified the following:
My repositories and contributions were visible previously.
I have not intentionally changed any visibility settings.
My repositories are public, and I have not deleted or archived my profile.
This issue started recently, and I suspect my account may have been flagged or restricted for some reason. If that is the case, I would greatly appreciate it if you could clarify the issue and guide me on the steps to resolve it.

My GitHub username is: KaleemNeslit

Please let me know if you need any additional information to assist with this request.

**Comment by ryanthedev99** [comment]
SAME THING IS HAPPENING TO ME >:(

**Comment by Abhishekmishra2808** [comment]
@ryanthedev99 after how many days, your profile came back?
